.rhdh/docs/openshift.md

```diff
@@ -0,0 +1,256 @@
+## Using the Operator to manage Red Hat Developer Hub (RHDH)
+
+### Installing RHDH using Operator
+
+**Prerequisites**
+
+- You are logged in using the OpenShift web console.
+- You have configured the appropriate roles and permissions within your project to create an application.
```

Comment:
In order to install the RHDP operator, the user must have a cluster admin role assigned.

Reply:
added suggestion to the previous line of the prereqs.